multisig/proposals/vip-012/vip-012-opbnbtestnet.ts

```diff
@@ -0,0 +1,52 @@
+import { NETWORK_ADDRESSES } from "../../../src/networkAddresses";
+import { makeProposal } from "../../../src/utils";
+
+const { opbnbtestnet } = NETWORK_ADDRESSES;
+
+const VTOKEN_BEACON = "0xcc633492097078Ae590C0d11924e82A23f3Ab3E2";
+const VTOKEN_IMPL = "0xd1fC255c701a42b8eDe64eE92049444FF23626A0";
+const VTOKEN_IMPL_TEMPORARY = "0xB8b90d642878C0561A85D96b540F7F0309293022";
+
+const vWBNB = "0xD36a31AcD3d901AeD998da6E24e848798378474e";
+const vWBNB_NEW_UNDERLYING = "0x4200000000000000000000000000000000000006";
+const STALE_PERIOD = 86400;
+
+export const vip012 = () => {
+  return makeProposal([
+    {
+      target: VTOKEN_BEACON,
+      signature: "upgradeTo(address)",
+      params: [VTOKEN_IMPL_TEMPORARY],
+    },
+    {
+      target: vWBNB,
+      signature: "setUnderlyingToken(address)",
+      params: [vWBNB_NEW_UNDERLYING],
+    },
+    {
+      target: VTOKEN_BEACON,
+      signature: "upgradeTo(address)",
+      params: [VTOKEN_IMPL],
+    },
+    {
+      target: opbnbtestnet.BINANCE_ORACLE,
+      signature: "setMaxStalePeriod(string,uint256)",
+      params: ["BNB", STALE_PERIOD],
```

Comment:
The symbol of the underlying token is WBNB

Reply:
FeedRegistry supports `BNB` as a qoute asset, we overidden symbol from `WBNB` to `BNB` previously see [here](https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/vip-001/vip-001-opbnbtestnet.ts#L122).

moreover, I think we don't need this step because the stale period for `BNB` is already configured.

Reply:
Removed the command https://github.com/VenusProtocol/vips/pull/224/commits/8a2851d8deba6509ec4d93fae78e89b1a80a9a6d